cmd/admin-info.go

```diff
@@ -115,7 +115,34 @@ func (u clusterStruct) String() (msg string) {
 		if srv.State == "offline" {
 			// "PrintB" is color blue in console library package
 			msg += fmt.Sprintf("%s  %s\n", console.Colorize("InfoFail", dot), console.Colorize("PrintB", srv.Endpoint))
-			msg += fmt.Sprintf("   Uptime: %s\n\n", console.Colorize("InfoFail", "offline"))
+			msg += fmt.Sprintf("   Uptime: %s\n", console.Colorize("InfoFail", "offline"))
+
+			if backendType != "FS" {
```

Comment:
I do not fully understand, if the server state is 'offline' (L115), why do we show/calculate online/offline disks in the first place ?

Reply:
Because to show which disks are offline @vadmeste since the server you are sending the request knows which disks belong to the host which is down.